client/state/sites/reducer.js

```diff
@@ -27,15 +29,18 @@ export function items( state = {}, action ) {
 				[ action.site.ID ]: action.site
 			} );
 		case SERIALIZE:
-			// scrub _events, _maxListeners, and other misc functions
+			//TODO: site objects are being decorated in SitesList in lib/sites/sites-list/index.js
+			//with either lib/site/index.js or lib/site/jetpack.js
```

Comment:
This is somewhat of a tangent, but I also checked what it would take to clean up sites in the redux tree. I ran into a few issues:
1) How do we cleanly pass the redux store to SitesList, where the api requests are occurring?
2) We expect our in memory sites to be class instances of lib/sites/index.js or lib/site/jetpack.js. Should we move away from this pattern?